summarize_pr.py

```diff
@@ -1,17 +1,15 @@
 import os
```

Comment:
The implementation of the `summarize_pr` function is well-structured and utilizes the Gemini model effectively. 

It would be beneficial to include error handling to gracefully manage cases where the GitHub API calls fail or the Gemini model returns unexpected results. This could involve implementing `try-except` blocks and logging errors for better debugging and user experience.